src/main.rs

```diff
@@ -1450,51 +1450,43 @@ fn get_paths_from_man_locations() -> Vec<PathBuf> {
 mod deroff;
 
 /// Generate fish completions from manpages.
-#[derive(StructOpt, Debug)]
-struct Opts {
+#[derive(Parser, Debug)]
+struct Args {
     /// Level of debug output.
-    #[structopt(short, long, default_value = "0", possible_values = &["0", "1", "2"])]
+    #[clap(
+        short, long, default_value = "0",
+        value_parser = clap::value_parser!(u8).range(0..3)
+    )]
```

Comment:
Nice! Is this evaluated at runtime or compile-time?

Reply:
Compile time I believe, but the error for this is kinda bad, I wish it is `[0, 1, 2]` but I didn't look closely.